packages/flutter_markdown/lib/src/widget.dart

```diff
@@ -546,25 +545,6 @@ class Markdown extends MarkdownWidget {
   }
 }
 
-/// Parse [task list items](https://github.github.com/gfm/#task-list-items-extension-).
-class TaskListSyntax extends md.InlineSyntax {
```

Comment:
widget.dart is fully exported, so this is a breaking change.

It should be `@Deprecated`, not removed.

Reply:
I have restored the `TaskListSyntax` class and added `@Deprecated` and a comment concerning it's replacement and future removal. 
Open to suggestions on my `@Deprecated()` message.. here I am suggesting the use of one of the new syntax classes which are now native in Markdown.
```
@Deprecated('Use [OrderedListWithCheckBoxSyntax] or [UnorderedListWithCheckBoxSyntax]')
```